pkg/injector/pod_patch.go

```diff
@@ -741,8 +741,8 @@ func getSidecarContainer(annotations map[string]string, id, daprSidecarImage, im
 		SecurityContext: &corev1.SecurityContext{
 			AllowPrivilegeEscalation: &allowPrivilegeEscalation,
 		},
-		Ports:   ports,
-		Command: cmd,
+		Ports: ports,
+		Args:  append(cmd, args...),
```

Comment:
Why this change?

Reply:
See here: https://github.com/dapr/dapr/pull/4741#discussion_r901261917

Reply:
10x

Reply:
Command overrides the entrypoint in the docker image. This change ensures that we can run our custom entrypoint scripts. If Command is passed, the entrypoint gets overridden. Dapr and its args, are now passed as Arguments.

On non windows container, the default entrypoint is used since we don't specify an explicit one.

Reply:
10x?

Reply:
> 10x?

ten + x = thanks